GPRegistryPolicyParser.psm1

```diff
@@ -185,14 +193,7 @@ Function Read-PolFile
     $index = 0
 
     [string] $policyContents = Get-Content $Path -Raw
-    if ( $PSVersionTable.PSVersion.Major -le "5")
-    {
-        [byte[]] $policyContentInBytes = Get-Content $Path -Raw -Encoding Byte
-    }
-    elseif ($PSVersionTable.PSVersion.Major -gt "5")
-    {
-        [byte[]] $policyContentInBytes= get-content $path -AsByteStream -Raw
-    }
```

Comment:
Instead of having to use version logic at every get/add-content, we can just splat the hashtable